migrate_master_item_reference_to_asset_relations.sql

```diff
@@ -0,0 +1,159 @@
+-- This script will migrate from an existing master item reference field to asset relations. 
+-- Be aware that it will remove any existing relations for the specified relation type.
+
+-- Set these variables
+declare @sourceMasterItemReferenceFieldItemGuid uniqueidentifier = '';
+declare @targetAssetRelationTypeId int = 0;
+
+-- Script starts here
+begin transaction;
+
+-- Verify metafield is a master item reference field
+if not exists (select *
+               from item_metafield imf
+                        join item i on imf.item_id = i.itemid
+               where i.ItemGuid = @sourceMasterItemReferenceFieldItemGuid
+                 and imf.item_datatypeid = 80
+                 and imf.autotranslateoverwriteexisting = 1)
+    begin
+        throw 51000, 'The specified metafield either does not exist, it is not a MasterItemReference field or it doesn''t have autotranslateoverwritingexisting enabled', 1;
```

Comment:
Because that way i don't have to deal with conflicting values between languages. Asset relations are not language versioned, while metadata is, so this was the easiest solution to that. I also think that all customers use the field this way, so it should be an "easy" requirement.